app-config.yaml

```diff
@@ -111,21 +110,7 @@ catalog:
 
   locations:
     - type: url
-      target: https://github.com/hmcts/template-expressjs/blob/master/template.yaml
-      rules:
-        - allow: [ Template ]
-    - type: url
-      target: https://github.com/hmcts/template-spring-boot/blob/master/template.yaml
-      rules:
-        - allow: [ Template ]
-    - type: url
-      target: https://github.com/hmcts/template-java-client/blob/master/template.yaml
-      rules:
-        - allow: [ Template ]
-    - type: url
-      target: https://github.com/hmcts/template-product-infrastructure/blob/master/template.yaml
-      rules:
-        - allow: [ Template ]
+      target: https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/backstage/labs-jackmaloney2-api.yaml
```

Comment:
we want to keep those templates that were there before, you prob don't want this added as a location either

Reply:
Done